aspnetcore/test/dev-tunnels.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Dev tunnels in Visual Studio 2022
+author: tdykstra
+description: Learn how to use dev tunnels in Visual Studio with ASPNET Core apps.
+monikerRange: '>= aspnetcore-7.0'
+ms.author: tdykstra
+ms.date: 12/28/2022
+uid: test/tunnels
+---
+# How to use dev tunnels in Visual Studio 2022 with ASP.NET Core apps
+
+The *Dev tunnels* feature of Visual Studio 2022 enables ad-hoc connections between machines that can't directly connect to each other. A URL is created that enables any device with an internet connection to connect to an ASP.NET Core project while it runs on localhost.
```

Comment:
Can we change "Dev tunnels" to "dev tunnels", the branding for this service is to have it lower case unless it's required to be upper case (for example, tool window names in VS use caps).